lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -92,6 +92,7 @@ class HttpApiEvents {
     const properties = {
       Name: this.provider.naming.getHttpApiName(),
       ProtocolType: 'HTTP',
+      DisableExecuteApiEndpoint: this.config.disableDefaultEndpoint,
```

Comment:
Let's do: `this.config.disableDefaultEndpoint == null ? undefined : this.config.disableDefaultEndpoint`

This will ensure that in case of `null` or `undefined` we assign `undefined` (which is automatically stripped), and only any other value is respected

Reply:
Thanks! I don't know much about JS. That looks good, I added it :)